Admin/AdminHelper.php

```diff
@@ -284,13 +285,11 @@ public function getElementAccessPath($elementId, $entity)
                     $partReturnValue .= $dot.$id;
                     $dot = '.';
```

Comment:
Is there still a need for this to be variable ? Its value is now `''`, then `'.'` forever, maybe a `SEPARATOR` constant could be used, or simply `'.'` ?